jujugui/static/gui/src/app/components/user-profile/model-list/model-list.js

```diff
@@ -222,6 +217,17 @@ YUI.add('user-profile-model-list', function() {
             switchModel={this.switchModel} />
         );
       }
+      const list = this.state.modelList;
+      let content;
+      if (list && list.length > 0) {
+        const rows = list.map(this._generateRow);
+        content = (
+          <ul className="user-profile__list twelve-col">
+            {this._generateHeader()}
+            {rows}
+          </ul>
+        );
+      }
       return (
```

Comment:
@anthonydillon It will return the model list header with the button when the model list is empty. If everything else (bundles, charms, budgets, terms, etc.) are also all empty, the empty user profile page will be displayed instead.